src/main/java/Duke.java

```diff
@@ -2,14 +2,34 @@
 
 public class Duke {
     public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String instructions = "Hello! Welcome to Duke. I am your personal task tracker.\n"
```

Comment:
I like that you put all the available functions in the instruction for the user! :)